var/spack/repos/builtin/packages/py-clang/package.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyClang(PythonPackage):
+    """Python bindings for clang from clang release branches"""
+
+    homepage = "https://clang.llvm.org/"
+    pypi     = "clang/clang-5-5.0.tar.gz"
```

Comment:
```suggestion
    pypi     = "clang/clang-5.0.tar.gz"
```

Reply:
That was a carryover from https://pypi.org/project/clang-5/5.0/. That was the dependency in the spec but then I switched to the main clang project and pulled version 5.